sample/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+<!--?xml version="1.0" encoding="UTF-8"?-->
```

Comment:
why is this commented out?

Reply:
That's a mistake.

Reply:
I used this [Color Tool](https://material.io/resources/color/#!/?view.left=0&view.right=0) to generate the right colors for me and missed that the color.xml it generated made that change.